libethploit/EVMAnalyser.cpp

```diff
@@ -61,15 +58,20 @@ bool EVMAnalyser::isEthploitModeEnabled() {
     return ethploitMode;
 }
 
+void EVMAnalyser::setBadTransaction() {
+    badTransaction = true;
+}
+
 void EVMAnalyser::setupTransaction(std::string account,
                                    std::string _transactionHash, 
                                    dev::u256 senderBalance, 
                                    dev::u256 receiverBalance,
                                    int64_t blockNumber) {
     if (senderBalance != -1 && receiverBalance != -1) {
         if (transactionHash != _transactionHash) {// New transaction
+            badTransaction = false;
             logger->setTransactionInfo(account, _transactionHash, blockNumber);
-
+            badTransaction = false;
```

Comment:
Why there is a duplicate line of setting badTransaction false?

Reply:
This is to reset for the next trx if the prev is set to true. If that's the duplication u mean